tasks/go.py

```diff
@@ -411,6 +411,16 @@ def tidy_all(ctx):
 
 @task
 def tidy(ctx):
+    if os.name != 'nt':
```

Comment:
What is `nt` ? Might be worth to add a small comment to explain that.

Reply:
Windows, adding it